Lib/asyncio/base_events.py

```diff
@@ -717,6 +717,8 @@ def call_at(self, when, callback, *args, context=None):
 
         Absolute time corresponds to the event loop's time() method.
         """
+        if when is None:
+            raise ValueError("when cannot be None")
```

Comment:
Arguably a TypeError?

Reply:
Agree with @gvanrossum 
`call_later()` should be updated as well.
Now it throws `TypeError: unsupported operand type(s) for +: 'float' and 'NoneType'`
The error text is very unclear.

Reply:
What should be updated in `call_later`?

Reply:
Please implement a check similar to added to `call_at()`:
```
if when is None:
    raise TypeError(...)
```

Reply:
Done